lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -228,6 +228,11 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
     }
   }
 
+  late final ViewFocusBinding _viewFocusBinding = ViewFocusBinding(
```

Comment:
```suggestion
  final ViewFocusBinding _viewFocusBinding = ViewFocusBinding(
```

It is not `late` when you're initializing it immediately, is it?

Reply:
It is late because I need the view manager (which is not static).